### PR TITLE
Skip over missing groups

### PR DESCRIPTION
**Pull request by jvillafanez**
Fix https://github.com/owncloud/user_ldap/issues/20

### CODE REVIEW

File: lib/Command/UpdateGroup.php [hunk 1]
```diff
@@ -196,25 +197,34 @@ private function updateGroupMapping($groupName, $userList) {
...
 			->where($query->expr()->in('owncloud_name', $query->createParameter('groups')))
 			->setParameter('groups', $groupNames, IQueryBuilder::PARAM_STR_ARRAY);
 		$result = $query->execute();
-		$row = $result->fetch();
 
-		$countValue = intval($row['ngroups']);
+		// fill with those which are inside the DB
+		while (($row = $result->fetch()) !== false) {
+			if (in_array($row['owncloud_name'], $groupNames, true)) {
+				$groups['inDB'][] = $row['owncloud_name'];
```
Comment (PVince81) [inline lib/Command/UpdateGroup.php]: I hope we won't run out of memory here with crazy setups that have thousands of groups

Comment (PVince81) [inline lib/Command/UpdateGroup.php]: Ah never mind, this is only based on the input list, so unless some admin passes 1000 groups to the CLI it should be fine.

**Comment by PVince81** [comment]
Tested, works :+1: 

Feel free to merge unless you want to change the message as advised.

**Comment by jvillafanez** [comment]
Ready to be merged

**Comment by jvillafanez** [comment]
Before we forget, this should be backported to 9.1 and 9.0.

**Comment by PVince81** [comment]
> Before we forget, this should be backported to 9.1 and 9.0.

Go go go !

**Comment by jvillafanez** [comment]
Backports:
* stable9.1 https://github.com/owncloud/core/pull/26402
* stable9: https://github.com/owncloud/core/pull/26403

**Pull request merged**